symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkClientConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.symphony.bdk.core.config.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BdkClientConfig {
+
+    private static final String DEFAULT_SCHEME = "https";
+    private static final int DEFAULT_HTTPS_PORT = 443;
+
+    private String scheme = DEFAULT_SCHEME;
+    private String url;
```

Comment:
does this mean we have to configure it like:

agent:
 url:
 port:
 scheme:
... ? it feels a bit cumbersome no?

Reply:
For example, we will have:
```
agent:
   url:
   port:
   scheme:
   context:
pod:
   url:
   port:
   scheme:
   context:
```
instead of in v1:
```
  agentHost:
  agentPort:
  agentContextPath:
  podHost:
  podPort:
  podContextPath:
```
I think it would be clearer and more consistent

Reply:
would not it be easier to just configure an URL?